src/main/resources/i18n/pt_BR.yml

```diff
@@ -0,0 +1,506 @@
+        helpKill: "{cc}/uh kill <player> {ci}: Assinala o jogador como morto, mesmo que ele esteja desconecato."
+        helpResurrect: "{cc}/uh resurrect <jogador> {ci}: Ressuscita o jogador."
+        helpTpback: "{cc}/uh tpback <jogador> [force] {ci}: Teletransporta em segurança um jogador para o local da sua morte. Quando especificado - force - teletransporta o jogador mesmo que não seja seguro."
+        
+        titleMiscCmd: "{aqua}------ Outros Comandos ------"
+        helpFreeze: "{cc}/uh freeze {ci}: (Des)congela o jogo todo, ou um jogador. Digite: /uh freeze para detalhes."
+        helpFinish: "{cc}/uh finish {ci}: Exibe o nome do(s) vencedor(es) e lança foguetes."
+        helpTP: "{cc}/uh tp {ci}: Teletransporta os espectadores ou uma equipe inteira. Digite /uh tp para detalhes."
+        helpTimers: "{cc}/uh timers {ci}: Configura os tempos. Digite /uh timers para detalhes."
+        helpInfos: "{cc}/uh infos {ci}: Printa somente as informações sobre o jogo atual."
+        helpAbout: "{cc}/uh about {ci}: Informação sobre o Plugin e as traduções."
+        
+        teamHelpTitle: "{aqua}------ Comandos para equipes ------"
+        teamHelpAdd: "{cc}/uh team add <cor> {ci}: Adiciona uma equipe com uma determinada cor."
+        teamHelpAddName: "{cc}/uh team add <cor> <nome ...> {ci}: Adiciona uma equipe com um determinado nome e cor. Podem ser usados espaços."
+        teamHelpRemove: "{cc}/uh team remove <nome ...> {ci}: Remove uma equipe"
+        teamHelpJoin: "{cc}/uh team join <jogador> <equipe ...>{ci}: Adiciona um jogador a uma equipe. O nome da equipe pode ser a cor ou o seu determinado nome."
+        teamHelpLeave: "{cc}/uh team leave <jogador> {ci}: Retira um jogador da sua equipe."
+        teamHelpList: "{cc}/uh team list {ci}: Lista das equipes e seus jogadores."
+        teamHelpReset: "{cc}/uh team reset {ci}: Elimina todas as equipes."
+        teamHelpJoinCmd: "{cc}/join [player] <team ...> {ci}: Adiciona “jogador” (ou ele mesmo) para a equipe determinada. Sem argumentos, apresenta opção escolher equipe via chat (clicando na equipe)."
+        teamHelpLeaveCmd: "{cc}/leave [player] {ci}: Remove o “jogador” (ou ele mesmo) de seu time."
+        
+        spawnsHelpTitle: "{aqua}------ Comandos para os Spawns ------"
+        spawnsHelpAdd: "{cc}/uh spawns add {ci}: Adiciona um local de spawn para uma equipe ou jogador, no local onde é executado."
+        spawnsHelpAddXZ: "{cc}/uh spawns add <x> <z> {ci}: Adiciona um local de spawn para uma equipe ou jogador, nas coordenadas inseridas, no mundo padrão."
+        spawnsHelpGenerate: "{cc}/uh spawns generate <circular|grid|random> [Tamanho = Tamanho dos Limites] [DistânciaMin = 250] [locais de spawn = Número de equipes] [xCentro = xSpawn] [zCentro = zSpawn] [world = mundo onde é executado o comando] {ci}: Gera automaticamente os locais de spawn com a forma inserida."
```

Comment:
The translation for this key is not correct (outdated). The current key to translate is:
```yml
spawnsHelpGenerate: "{cc}/uh spawns generate {ci}: automagically generates spawn points. See /uh spawns generate for details."
```